backend/src/main/java/com/htecgroup/skynest/exception/auth/UnableToChangeOtherUserPasswordException.java

```diff
@@ -0,0 +1,12 @@
+package com.htecgroup.skynest.exception.auth;
+
+import org.springframework.http.HttpStatus;
+
+public class UnableToChangeOtherUserPasswordException extends AuthException {
```

Comment:
Maybe rename to PasswordChangeForbiddenException.. the message inside tells you the reason